### PR TITLE
Introduce issue forms

### PR DESCRIPTION
**Pull request by anesabml**
## What do these changes do?

Introduce [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)

## Are there changes in behavior for the user?

The new issue forms will allow contributors to include specific, structured information.

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is `<Name> <Surname>`.
  * Please keep alphabetical order, the file is sorted by names.
- [x] Add a new news fragment into the [`CHANGES/`](../tree/master/CHANGES) folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example:
   `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

**Review by Andrew-Chen-Wang — CHANGES_REQUESTED** [review]
@anesabml please run the pre-commit linter to fix the yml fixer. Can I ask: did you write this yourself or copy this from a different repo. I can't really verify it works since forks don't have issues.

**Comment by anesabml** [comment]
@Andrew-Chen-Wang you can take a look at [aiohttp](https://github.com/aio-libs/aiohttp) issue form.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+name: Bug Report
+description: Create a report to help us improve.
+labels: [bug]
+assignees: aio-libs/triagers
```
Comment (Andrew-Chen-Wang) [inline .github/ISSUE_TEMPLATE/bug_report.yml:5]: I don't know if any of the maintainers here are in that group.
```suggestion
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+contact_links:
+- name: 🤷💻🤦 StackOverflow
+  url: https://stackoverflow.com/questions/tagged/aioredis
+  about: Please ask typical Q&A here
+- name: 💬 Discourse
+  url: https://aio-libs.discourse.group/
+  about: Please start usage discussions here
+- name: 💬 Gitter Chat
+  url: https://gitter.im/aio-libs/Lobby
+  about: Chat with devs and community
```
Comment (Andrew-Chen-Wang) [inline .github/ISSUE_TEMPLATE/config.yml:12]: I'm not really on any of these three; I stick to GitHub issues (or maybe the Discussion tab). Idk cc @seandstewart @abrookins thoughts on this contact links?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+- type: textarea
+  attributes:
+    label: To Reproduce
+    description: >-
+      Describe the steps to reproduce this bug.
+    placeholder: |
+      1. Have certain environment
+      2. Run given code snippet in a certain way
+      3. See some behavior described
```
Comment (Andrew-Chen-Wang) [inline .github/ISSUE_TEMPLATE/bug_report.yml:37]: Had to re-read this a couple of times. "Have certain environment" could be rewritten to "Your environment (OS, etc.)". But yea please expand on this; could be confusing at first.

**Review by Andrew-Chen-Wang — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (anesabml) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: I agree, I've changed the placeholder to:
```
1. Create a certain environment (OS, Environment Variables, etc)
2. Run a code snippet '...'
3. An error occurs
```
is that clear enough or should I think of another placeholder?

Comment (Andrew-Chen-Wang) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Maybe 3. Describe the error that is occurring

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (anesabml) [reply .github/ISSUE_TEMPLATE/config.yml:12]: Should I remove this file?

**Review by Andrew-Chen-Wang — APPROVED** [review]
Thanks!

**Pull request merged**